src/test/java/org/ays/auth/tests/UsersListTest.java

```diff
@@ -195,6 +202,90 @@ public void listUsersForInvalidFirstAndLastNameValue(String firstName, String la
+
+        UserListPayload userListPayload = new UserListPayload();
+        userListPayload.setPageable(AysPageable.generate(1, 10));
+
+        UserListPayload.Filter filter= new UserListPayload.Filter();
+        filter.setEmailAddress(emailAddress);
+
+        List<String> statuses = new ArrayList<>();
+        statuses.add("ACTIVE");
+        statuses.add("PASSIVE");
+        filter.setStatuses(statuses);
+
+        userListPayload.setFilter(filter);
+
+        Response response=UserEndpoints.findAll(userListPayload,accessToken);
+        response.then()
+                .spec(AysResponseSpecs.expectSuccessResponseSpec())
+                .body("response.content[0].emailAddress",equalTo(emailAddress))
+                .body("response.content[0].id",notNullValue());
+
+    }
+
+    @Test(groups = {"Regression"})
+    public void listAllUsersByEmailAddress(){
+
+        LoginPayload loginPayload = LoginPayload.generateAsTestVolunteerFoundationAdmin();
+        String accessToken = this.loginAndGetAccessToken(loginPayload);
+
+        UserListPayload userListPayload = new UserListPayload();
+        userListPayload.setPageable(AysPageable.generate(1, 10));
+
+        UserListPayload.Filter filter= new UserListPayload.Filter();
+        filter.setEmailAddress("@gmail.com");
+
+        List<String> statuses = new ArrayList<>();
+        statuses.add("ACTIVE");
+        statuses.add("PASSIVE");
+        filter.setStatuses(statuses);
+
+        userListPayload.setFilter(filter);
+
+        Response response=UserEndpoints.findAll(userListPayload,accessToken);
+        response.then()
+                .spec(AysResponseSpecs.expectSuccessResponseSpec())
+                .body("response.content.size()",greaterThan(0))
+                .body("response.content[0].id",notNullValue())
+                .body("response.content[0].firstName",notNullValue())
+                .body("response.content[0].emailAddress",notNullValue());
+
+    }
+
+    @Test(groups = {"Regression"},dataProvider = "invalidEmailAddressForUsersList", dataProviderClass = AysDataProvider.class)
+    public void listUsersForInvalidEmailAddressData(String emailAddress, AysErrorMessage errorMessage, String field, String type){
+
+        LoginPayload loginPayload=LoginPayload.generateAsTestVolunteerFoundationAdmin();
+        String accessToken=this.loginAndGetAccessToken(loginPayload);
+
+        UserListPayload userListPayload= new UserListPayload();
+        userListPayload.setPageable(AysPageable.generate(1,10));
+
+        UserListPayload.Filter filter= new UserListPayload.Filter();
+        filter.setEmailAddress(emailAddress);
+        userListPayload.setFilter(filter);
+
+        Response response=UserEndpoints.findAll(userListPayload,accessToken);
+        response.then()
+                .spec(AysResponseSpecs.expectBadRequestResponseSpec())
+                .spec(AysResponseSpecs.subErrorsSpec(errorMessage,field,type));
+
+    }
+
```

Comment:
Bu sınıf formatlanmamış gibi görünüyor. IntellIJ içerisinde otomatik formatlamayı aktifleştirmek için aşağıdaki dokümandan faydalınabilir;
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html